ansible/roles/ooni-explorer-next/defaults/main.yml

```diff
@@ -1,3 +1,4 @@
 explorer_next_tag: '20181112-00674a72'
 explorer_next_backend_ipv4: 172.27.33.254
 explorer_next_backend_port: 3100
+explorer_country_data_path: /var/www/explorer
```

Comment:
Done. Do I also have to setup copying all the files in the directory from somewhere? Or is this just to prevent nginx from trying to serve from a non-existent directory somehow?

Reply:
I think it's fine that you do that manually as that is part of the development process. The reason to create the dir as part of the playbook is to ensure that on a clean machine it will do everything that is necessary to get it to work (manually settings permissions or creating directories can be error prone).